sdk/README.md

```diff
@@ -42,12 +42,33 @@ Sample code for the application which send out data to others.
 
 Source code: samples/sender_app.c
 ```bash
-$ ./build/samples/sender_app
-usage: sender_app [OPTION]
--h, --help                      Print this help and exit.
--s, --ip=ip_address             Send data to IP address (default: 127.0.0.1).
--p, --port=port_number          Send data to Port (default: 9001).
--n, --number=frame_number       Total frame number to send (default: 300).
+Usage: sender_app [OPTION]
+-H, --help                              Print this help and exit
+-r, --rcv_ip=<ip_address>               Receiver's IP address (default: 127.0.0.1)
+-i, --rcv_port=<port_number>            Receiver's port number (default: 9001)
+-s, --send_ip=<ip_address>              Sender's IP address (default: 127.0.0.1)
+-p, --send_port=<port_number>           Sender's port number (default: 9001)
+-o, --protocol=<protocol_type>          Set protocol type (default: auto)
+-t, --type=<payload_type>               Payload type (default: st20)
+-k, --socketpath=<socket_path>          Set memif socket path (default: /run/mcm/mcm_rx_memif.sock)
+-m, --master=<is_master>                Set memif conn is master (default: 1 for sender, 0 for recver)
+-d, --interfaceid=<interface_id>        Set memif conn interface id (default: 0)
+-b, --file=<input_file>                 Input file name (optional)
+-l, --loop=<is_loop>                    Set infinity loop sending (default: 0)
+--------------------------------------   VIDEO (ST2x)   --------------------------------------
+-w, --width=<frame_width>               Width of test video frame (default: 1920)
+-h, --height=<frame_height>             Height of test video frame (default: 1080)
+-f, --fps=<video_fps>                   Test video FPS (frame per second) (default: 30.00)
+-x, --pix_fmt=<pixel_format>            Pixel format (default: yuv422p10le)
+-n, --number=<number_of_frames>         Total frame number to send (default = inf: 0)
```

Comment:
```suggestion
-n, --number=<number_of_frames>         Total frame number to send (default: 0, i.e. infinite)
```

Reply:
It is to emphasize that default value is 0, which means infinite.

Reply:
Not sure if the user understands what `inf` is. I suggest changing to `default: 0, i.e. infinite`

Reply:
Don't get me wrong, "id est" fits here the best, but I'd rather stick to `default = infinite: 0`. I simply know "i.e." is too often wrongly understood to be equivalent of "e.g.".

Reply:
Agree